recipes/sktools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sktools" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dftbplus/skprogs/archive/refs/tags/{{ version }}.tar.gz
+  sha256: e63e6a38f920bbb85f1f8fdff2da5ec195a3f2ded354f6f09cf68cf69785b59d
+
+build:
+  number: 0
+  noarch: python
```

Comment:
This is not a noarch package because it uses the scripts argument in setup.py.

After switching to non-noarch you may need to patch the install so the scripts are installed to the correct place for windows environments. I'm not sure.

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Reply:
The consuming package is currently not Windows compatible, I'll try to get upstream there, but no promises.

Reply:
Raised in https://github.com/dftbplus/skprogs/issues/35